hypershift-operator/controllers/hostedcluster/metrics/metrics.go

```diff
@@ -185,6 +188,14 @@ var (
 	clusterSizeOverrideMetricDesc = prometheus.NewDesc(
 		ClusterSizeOverrideMetricName, clusterSizeOverrideMetricHelp,
 		append(hclusterLabels, "environment", "internal_id", "size"), nil)
+
+	hostedClusterInfoDesc = prometheus.NewDesc(
+		HostedClusterInfoMetricName, HostedClusterInfoMetricHelp,
+		append(hclusterLabels, "Microsoft_subscriptionId",
```

Comment:
@tony-schndr does it make sense to include other fields of the Azure's HostedCluster spec?
* Cloud
* Location
* VnetID
* SubnetID
* SecurityGroupID
* TenantID

Reply:
We need Location, I thought I saw that is already included?  TenantId might also be good to have though it is not required by SRE or Geneva Shoebox.

At the end of the day, everything else is nice to have but SRE has other means to get that information.  The resource ID / region allows us to look everything else up from the RP.

Reply:
@simonpasquier @tony-schndr thanks for this! It really help? May you this to the https://issues.redhat.com/browse/OCPSTRAT-2175 ? 
Eventually what about changing to `snake_case` as suggested by @devguyio https://github.com/openshift/hypershift/pull/6180#discussion_r2108610241
Ty!